tests/test_zenroom.py

```diff
@@ -92,7 +93,15 @@ def test_zenroom():
     print("============== PUBLIC IDENTITIES =================")
     print(zen_public_keys)
 
+    # the result key is the expected result of the fulfill script
+    # it depends on the script, in this case I know that
+    #     `Then print the string 'ok'`,
+    # results in
+    #     { "output": ["ok"] }
     metadata = {
+            "result": {
+                "output": ["ok"]
+            }
     }
```

Comment:
@albertolerda why do we need the metadata defined?
it's now mandatory for all integrations. why did you add this requirement?